cdap-ui/app/cdap/components/FieldLevelLineage/v2/LineageSummary/index.tsx

```diff
@@ -48,6 +50,7 @@ const styles = (theme): StyleRules => {
       justifyContent: 'space-between',
       position: 'relative',
       overflowX: 'hidden',
+      overflowY: 'hidden',
```

Comment:
is this overflowY hidden intended? ie, is the root will always be 100% of the height?

If so, then could probably combine this with line 52 to `overflow: hidden`

Reply:
Good call, I've combined them!